src/simulation/estimation_setup/observationOutput.cpp

```diff
@@ -22,10 +22,10 @@ void checkObservationDependentVariableEnvironment(
         const SystemOfBodies& bodies,
         const std::shared_ptr< StationAngleObservationDependentVariableSettings > variableSettings )
 {
-    if( isObservationDependentVariableGroundStationProperty( variableSettings ) && variableSettings->isLinkEndDefined_ )
+    if( isObservationDependentVariableGroundStationProperty( variableSettings->variableType_ ) && variableSettings->isLinkEndDefined_ )
```

Comment:
Should this function (checkObservationDependentVariableEnvironment) not be extended for other dependent variables? Right now it only checks the station angle variables

Reply:
good point, I didn't extend this function indeed